package.json

```diff
@@ -68,41 +67,30 @@
     "wreck": "^5.4.0"
   },
   "devDependencies": {
+    "angular": "^1.5.5",
+    "angular-animate": "^1.5.5",
+    "angular-cache": "jmdobry/angular-cache",
+    "angular-cookies": "^1.5.5",
+    "angular-gettext": "rubenv/angular-gettext",
+    "angular-sanitize": "^1.5.5",
+    "angular-ui-router": "^0.2.18",
+    "babel-core": "^6.8.0",
+    "babel-loader": "^6.2.4",
+    "babel-preset-es2015": "^6.6.0",
+    "bcapp-pattern-lab": "bigcommerce-labs/bcapp-pattern-lab",
+    "clean-webpack-plugin": "^0.1.9",
     "code": "^1.4.0",
+    "copy-webpack-plugin": "^2.1.3",
+    "es6-micro-loader": "^0.2.1",
+    "expose-loader": "^0.7.1",
+    "js-cookie": "^2.1.1",
+    "jschannel": "meenie/jschannel",
     "lab": "^5.5.1",
-    "sinon": "^1.14.1"
-  },
-  "jspm": {
-    "directories": {
-      "baseURL": "public"
-    },
-    "dependencies": {
-      "angular": "github:angular/bower-angular@1.3.7",
-      "angular-animate": "github:angular/bower-angular-animate@1.3.7",
-      "angular-cookies": "github:angular/bower-angular-cookies@1.3.7",
-      "angular-formly": "github:formly-js/angular-formly@7.3.3",
-      "angular-sanitize": "github:angular/bower-angular-sanitize@1.3.7",
-      "angular-ui-router": "github:angular-ui/ui-router@0.2.13",
-      "bigcommerce-labs/bcapp-pattern-lab": "github:bigcommerce-labs/bcapp-pattern-lab@1.17.6",
-      "bigcommerce-labs/ng-common": "github:bigcommerce-labs/ng-common@2.5.1",
-      "bigcommerce-labs/ng-stencil-editor": "github:bigcommerce-labs/ng-stencil-editor@1.7.1",
-      "jmdobry/angular-cache": "github:jmdobry/angular-cache@3.2.4",
-      "js-cookie": "github:js-cookie/js-cookie@^2.0.3",
-      "kentcdodds/api-check": "github:kentcdodds/api-check@^7.5.0",
-      "lodash": "npm:lodash@2.4.1",
-      "meenie/jschannel": "github:meenie/jschannel@0.0.5",
-      "ng-idle": "npm:ng-idle@1.1.1",
-      "rubenv/angular-gettext": "github:rubenv/angular-gettext@1.1.4"
-    },
-    "devDependencies": {
-      "babel": "npm:babel-core@^5.1.13",
-      "babel-runtime": "npm:babel-runtime@^5.1.13",
-      "core-js": "npm:core-js@^0.9.4"
-    },
-    "overrides": {
-      "github:bigcommerce-labs/bcapp-pattern-lab@1.17.6": {
-        "format": "global"
-      }
-    }
+    "ng-common": "bigcommerce-labs/ng-common",
+    "ng-stencil-editor": "bigcommerce-labs/ng-stencil-editor",
```

Comment:
Does this work? Don't you have to give it the github ssh url and also point it to a specific version?